index.html

```diff
@@ -191,14 +191,14 @@
         <p>To execute multiple statements, use a block statement ({ ... }) to group those statements.</p>
         <p>Example:</p>
         <p>The following while loop iterates as long as n is less than three:</p>
-        <code>var n = 0; var x = 0; while (n < 3) { n++; x += n; }</code>
+        <code>var n = 0; var x = 0; while (n &lt; 3) { n++; x += n; }</code>
         <p>With each iteration, the loop increments n and adds that value to x. Therefore, x and n take on the following values:</p>
         <ul>
```

Comment:
Another little thing to change  😃 
- **Element “ul” not allowed as child of element “ul” in this context.** You'll notice that you have <ul> elements wrapped inside other ul elements in some sections of your code (Check _Data types_ section), something that needs just a little tweaking to meet coding standards.
> You can find some insightful methods on how to use best practices for these [here](https://www.w3schools.com/html/html_lists.asp) or from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li)